src/lib/config.ts

```diff
@@ -55,7 +55,7 @@ const config: UserServiceConfig = {
   authenticationCookieName: ".AUTH",
   baseUrl: process.env.BASE_URL,
   bichardRedirectURL: process.env.BICHARD_REDIRECT_URL ?? "/bichard-ui/InitialRefreshList",
-  contactUrl: "https://mojprod.service-now.com/",
+  contactUrl: `${process.env.BASE_URL}/users/faq`,
```

Comment:
Can we just make this just use a relative URL since Bichard can be accessed via a number of different domains

Reply:
It can't be relative because the link is included in some of the email templates.

It should probably use the same `addBaseUrl` logic that the other links in emails use, though...

Or perhaps we should just link directly to the service desk URL from the emails, and only show the FAQ page when links within the user-service are clicked?